types/mapbox-gl/index.d.ts

```diff
@@ -421,6 +421,8 @@ declare namespace mapboxgl {
 
         resetNorth(options?: mapboxgl.AnimationOptions, eventData?: mapboxgl.EventData): this;
 
+        resetNorthPitch(options?: mapboxgl.AnimationOptions, eventData?: mapboxgl.EventData): this;
```

Comment:
Small thing here: "?" in TS means "T | undefined", but in Flow (which is used by mapbox) it means "T | undefined | null", so we should reflect that fact. I know, it has not been very consistent across this @types/mapbox-gl project from the beginning, but let's try to fix this over time.

Reply:
What would be a more reasonable approach:
1. adding `| null` i.e. `options?: mapboxgl.AnimationOptions | null`
or
2. creating some sort of new type that allows nullable i.e. `export type NullableAnimationOptions = mapboxgl.AnimationOptions | null;`

Reply:
I'd go for #1. That's what already been done in several places, I believe.